assets/src/theme/VarPicker.js

```diff
@@ -1,14 +1,36 @@
 import { useEffect, useState } from 'react';
 import { VariableControl } from './VariableControl';
-import { colorToState } from './colorToState';
 import { THEME_ACTIONS, useThemeEditor } from './useThemeEditor';
 import { whileHoverHighlight } from './highlight';
 import { exportCss, exportJson } from './export';
+import { useServerThemes } from './useServerThemes';
 
 export const LOCAL_STORAGE_KEY = 'p4-theme';
 
 const byName = (a, b) => a.name > b.name ? 1 : (a.name === b.name ? 0 : -1);
 
+const diffThemes = (themeA, themeB) => {
+  const added = Object.keys(themeB).filter(k=> !themeA[k]);
+  const removed = Object.keys(themeA).filter(k=> !themeB[k]);
+  const changed = Object.keys(themeB).filter(k=> !!themeA[k] && themeA[k] !== themeB[k]);
+  const hasChanges = added.length > 0 || removed.length > 0 || changed.length > 0;
+
+  return {added,removed,changed, hasChanges}
+}
+
+const diffSummary = (themeA, themeB) => {
```

Comment:
This is just a quick and dirty way to put the summary in a string and show it on hover using `title`.